tests/server/3-document.spec.js

```diff
@@ -100,7 +100,13 @@ describe('Document', () => {
           .set('authorization', adminToken)
           .end((err, res) => {
               res.status.should.equal(200);
-              res.body[1].should.have.value('role', 'regular');
+              for(let i=0; i<res.body.length; i++) {
+                  if (res.body[i].role === 'admin') {
```

Comment:
### Critical

This does not make sense since what you should be testing is if all roles are infact `regular` as specified in your query.